CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## [2.1.0] - 2022-04-25
```

Comment:
Just curious, is there a reason/pattern why you pushed the date forward? Most times when I update the changelog, I'm always torn between making the update during the review process or after review is finished so that the date will match the date of release also.

Reply:
Haha, yeah, I'm also trying to match the actual release date with what's written here in the change log. I was thinking of making the GitHub + PyPI release next Monday, so that in case there's a bug or something then I'll have a whole week to plan and fix things. Right now it's already Wednesday and we have other things to get to this week, and so I'm gonna merge this PR to master without immediately making a release just yet.